### DISCUSSION TITLE
llama_7b_8int认为自己是chatGPT

### DISCUSSION THREAD

**Discussion by No-22-Github** [ymcui/Chinese-LLaMA-Alpaca#329] [General]
![C}88X8Y_RRF1OI(CDTY5$D9](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/132265925/ba9ce1c4-2341-41bf-9598-81789729574a)
如图，角色扮演功能几乎失效，并且认为自己是chatgpt

**Comment by No-22-Github** [comment]
是LLaMA-Alpaca-7B-8int模型

**Comment by ymcui** [comment]
1、指令精调数据里没有包含“扮演”类的数据，如有需要可以自己精调。
2、同样，目前指令数据里包含了大量ChatGPT爬取数据，所以自我认同为ChatGPT没什么毛病。
3、看到你修改了prompt模板（默认应使用Alpaca模板），具体使用方法还需自行探索。

**Comment by No-22-Github** [comment]
好的，顺便问一下，精调Alpaca-7B-8int大概需要多少内存

**Comment by ymcui** [comment]
7B模型原始大小为13G（中文会稍大一点）。
按照社区用户的反馈，需要4090（24G）或以上的设备，具体跟选用的超参、训练的参数量等有关。
